### PR TITLE
Add filters to org search

### LINKED ISSUE #67: Add categories for orgs
Add types for orgs such as "academic", "cultural", "performance"

### LINKED ISSUE #68: Add filters for org search
filter with the newly added org types

### PR DESCRIPTION
**Pull request by lbourret**
-Add org type to organizations fixes #67 
-Add buttons to select multiple org types fixes #68

### CODE REVIEW

File: capstone/src/main/java/com/step902020/capstone/OrganizationRepository.java [hunk 1]
```diff
@@ -17,6 +19,11 @@
 
   public List<Organization> findByUniversity(University university);
 
+  public List<Organization> findByUniversityAndOrgType(University university, String orgType);
+
+  @Query("select * from organization where orgType = @orgType and university = @university")
+  public List<Organization> findOrganizationsByOrgType(@Param("orgType") String orgType, @Param("university") University university);
```
Comment (meltsufin) [inline capstone/src/main/java/com/step902020/capstone/OrganizationRepository.java:25]: You don't seem to be using this method.

File: capstone/src/main/resources/application.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
 ## Uncomment this line and insert your own json key file for local testing
-#spring.cloud.gcp.storage.credentials.location=file:/home/jennysheng/service-account.json
+spring.cloud.gcp.storage.credentials.location=file:/home/lbourret/service_account.json
```
Comment (meltsufin) [inline capstone/src/main/resources/application.properties:2]: Oops

When I run the app I don't even touch this file.
My command:
```
mvn clean package spring-boot:run  -Plocal -DskipTests -Dspring-boot.run.jvmArguments="-Dspring.cloud.gcp.storage.credentials.location=file:../../../step_service_account.json"
```

File: capstone/src/main/resources/templates/profile.html [hunk 3]
```diff
@@ -73,6 +73,18 @@
                     </select>
                     <label>Description:</label>
                     <input type="text" name="description" id="org-description" placeholder="Description of your organization.."></input>
+                    <label>Organization Type:</label>
+                    <select name="org-type" id="org-type">
+                        <option value="" disabled selected hidden>Choose organization type</option>
+                        <option value="academic">Academic</option>
```
Comment (meltsufin) [inline capstone/src/main/resources/templates/profile.html:79]: You can avoid hard-coding and duplicating the list of organization types by either creating an array in JavaScript shared across the two pages or storing  the types in a special OrgType table in Datastore.

**Comment by lbourret** [comment]
> I encountered this error where when I filter by arts there was suppose to be two organizations, but only one showed up. I don't know if this is a data problem or an actual bug. Other than that everything looks good!

It's because the two organizations are affiliated with different colleges

**Pull request merged**